### ISSUE TITLE
[procstat] Include information about number of cores

### ISSUE THREAD

**Issue by ptitou** [influxdata/telegraf#2020]
## Feature Request

When I use cpu_usage from procstat with a multicore system, cpu percentage can be greater than 100%. 
example on a 8core system and a process consuming 2 cpu : 
- on top global cpu reporting with cpu
- on bottom, cpu reporting for a spécific process that consume 2 cores
![image](https://cloud.githubusercontent.com/assets/16797555/20134571/2b2653e2-a66c-11e6-8cbb-438f2609b28d.png)

### Proposal:

Adding information of number of core on the system in order to make a graph with maxi 100% of cpu
![image](https://cloud.githubusercontent.com/assets/16797555/20134718/c2b4abb4-a66c-11e6-851e-1aaa57f025c8.png)

**Comment by KostyaSha** [comment]
The same is must have for load_average devision to get single load variables. 
ie. load average 15 means different for different number of cores. 

Any progress for it?

**Comment by danielnelson** [comment]
It occurs to me that you could get the number of cpus by enabling the cpu plugin and performing a query like this:
```
show tag values cardinality from "cpu" with key = "cpu" where "cpu" != 'cpu-total' and host = 'myhost';
```

To do the division in a single query I think you would need [ifql](https://github.com/influxdata/ifql).  We will probably not add a num cpu field to all measurements with fields that require knowing it to interpret.

**Comment by UnAfraid** [comment]
> It occurs to me that you could get the number of cpus by enabling the cpu plugin and performing a query like this:
> 
> ```
> show tag values cardinality from "cpu" with key = "cpu" where "cpu" != 'cpu-total' and host = 'myhost';
> ```
> 
> To do the division in a single query I think you would need [ifql](https://github.com/influxdata/ifql). We will probably not add a num cpu field to all measurements with fields that require knowing it to interpret.

This query worked fine in Grafana/influxdb-cli, but i cannot get it to work in Chronograf, i tried:
`SHOW TAG VALUES CARDINALITY FROM "cpu" WITH KEY="cpu" WHERE "cpu" != 'cpu-total' AND "host"=:host:` returns no results
`SHOW TAG VALUES CARDINALITY FROM "telegraf"."autogen"."cpu" WITH KEY="cpu" WHERE "cpu" != 'cpu-total' AND "host"=:host:` returns empty string
`SHOW TAG VALUES CARDINALITY ON "telegraf" FROM "cpu" WITH KEY="cpu" WHERE "cpu" != 'cpu-total' AND "host"=:host:` returns empty string

What do i do wrong?

**Comment by danielnelson** [comment]
@UnAfraid I'm not sure what the issue might be, could you open a [new issue with Chronograf](https://github.com/influxdata/chronograf/issues/new)?

**Comment by UnAfraid** [comment]
> @UnAfraid I'm not sure what the issue might be, could you open a [new issue with Chronograf](https://github.com/influxdata/chronograf/issues/new)?

Done: https://github.com/influxdata/chronograf/issues/5237

**Comment by glinton** [comment]
Regarding the original request: would it suffice to add a flag to turn irix mode off ([`cpu_perc`](https://github.com/influxdata/telegraf/blob/1.11.3/plugins/inputs/procstat/procstat.go#L236) would be divided by `float64(runtime.NumCPU())` before setting the `cpu_used` field)?

**Comment by UnAfraid** [comment]
> Regarding the original request: would it suffice to add a flag to turn irix mode off ([`cpu_perc`](https://github.com/influxdata/telegraf/blob/1.11.3/plugins/inputs/procstat/procstat.go#L236) would be divided by `float64(runtime.NumCPU())` before setting the `cpu_used` field)?

That would be lovely

**Comment by sjwang90** [comment]
Fixed in #6165
